### ISSUE TITLE
The downloaded video files using XDM are choppy and skip frames. (despite being of normal quality in the original files).

### ISSUE THREAD

**Issue by lalin1c** [subhra74/xdm#1094]
Previously, I was able to download videos with normal file quality.
About a month ago, I downloaded videos from the original source with normal file quality. 
However, the downloaded files have issues with video playback, including choppy, jumpy, or jerky and skip frames. 
I am using a Macbook Pro M2 macOS Ventura 13.1
XDM For MacOS Version 7.2.8
Download Video from JW Player 8.8.2

How should I go about resolving this problem?

Thank you in advance for your guidance.

**Comment by dsghi** [comment]
This may not be a file problem, but your playback app. If you're using Apple's default, you may want to try a third-party player like [VLC](https://www.videolan.org/vlc/download-macosx.html)

**Comment by lalin1c** [comment]
> This may not be a file problem, but your playback app. If you're using Apple's default, you may want to try a third-party player like [VLC](https://www.videolan.org/vlc/download-macosx.html)


@dsghi 
Yes, I opened the video file with VLC Player, but the video still skips frames and stutters. 

The original downloaded video file is in .ts format. 
I converted the downloaded file to .mp4 using Handbrake. 
https://handbrake.fr/rotation.php?file=HandBrake-1.5.1.dmg

However, the video still continues to skip frames and stutter as before.

**Comment by Swarthyp** [comment]
i have the same experience here on  Windows pc , did you get any help?
